src/pages/ProcessDetail.tsx

```diff
@@ -123,7 +123,7 @@ class ProcessDetail extends React.PureComponent<IProps, IState> {
         let enrichedProcess = processInstance as ProcessWithDetails;
 
         const stepUserInput: InputForm | undefined = enrichedProcess.form;
-        const tabs = stepUserInput ? this.state.tabs : ["process"];
+        const tabs = stepUserInput ? ["user_input", "process"] : ["process"];
```

Comment:
What does this change do?

Reply:
this is so when the `state.tabs` changes `["process"]` it doesnt become ` stepUserInput ? ["process"] : ["process"];`